packages/kbn-search-connectors/types/connectors.ts

```diff
@@ -6,6 +6,8 @@
  * Side Public License, v 1.
  */
 
+export type ConnectorType = 'elastic-crawler' | 'connector';
+export type ConnectorMode = 'native' | 'connector_client';
```

Comment:
question: why do we need this type? We already have an `is_native` boolean on connectors that does this. Adding this type just confuses matters, particularly because it's hard to parse the ConnectorType vs ConnectorMode at first glance